docs/docs/guides/integrate/login-ui/passkey.mdx

```diff
@@ -0,0 +1,308 @@
+Also you can give the Passkey a name, which makes it easier for the user to identify the registered authentication methods.
+Example: Google Pixel, iPhone, Mac Book, etc
+
+[Verify Passkey Registration Documentation](/apis/resources/user_service/user-service-verify-passkey-registration)
+
+```bash
+curl --request POST \
+  --url https://$ZITADEL_DOMAIN/v2alpha/users/$USER_ID/passkeys/$PASSKEY_ID \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer '"$TOKEN"''\
+  --header 'Content-Type: application/json' \
+  --data '{
+  "publicKeyCredential": {
+    "type": "public-key",
+    "id": "pawVarF4xPxLFmfCnRkwXWeTrKGzabcAi92LEI1WC00",
+    "rawId": "pawVarF4xPxLFmfCnRkwXWeTrKGzabcAi92LEI1WC00",
+    "response": {
+      "attestationObject": "o2NmbXRmcGFja2VkZ2F0dFN0bXSiY2FsZyZjc2lnWEcwRQIgRKS3VpeE9tfExXRzkoUKnG4rQWPvtSSt4YtDGgTx32oCIQDPey-2YJ4uIg-QCM4jj6aE2U3tgMFM_RP7Efx6xRu3JGhhdXRoRGF0YVikSZYN5YgOjGh0NBcPZHZgW4_krrmihjLHmVzzuoMdl2NFAAAAADju76085Yhmlt1CEOHkwLQAIKWsFWqxeMT8SxZnwp0ZMF1nk6yhs2m3AIvdixCNVgtNpQECAyYgASFYIMGUDSP2FAQn2MIfPMy7cyB_Y30VqixVgGULTBtFjfRiIlggjUGfQo3_-CrMmH3S-ZQkFKWKnNBQEAMkFtG-9A4zqW0",
+      "clientDataJSON": "eyJ0eXBlIjoid2ViYXV0aG4uY3JlYXRlIiwiY2hhbGxlbmdlIjoiQlhXdHh0WGxJeFZZa0pHT1dVaUVmM25zby02aXZKdWw2YmNmWHdMVlFIayIsIm9yaWdpbiI6Imh0dHBzOi8vbG9jYWxob3N0OjgwODAifQ"
+    }
+  },
+  "passkeyName": "fido key"
+}'
+```
+
+You have successfully registered a new passkey to the user.
+Next step is to authenticate the user with the new registered passkey.
+
+## Create Session
+
+First step is to ask the user for his username and create a new session with the ZITADEL API.
+When creating the new session make sure to include the challenge for passkey.
+The response will include the public key credential request options for the passkey in the challenges.
+
+[Create Session Documentation](/apis/resources/session_service/session-service-create-session)
+
+### Request
+```bash
+curl --request POST \
+  --url https://$ZITADEL_DOMAIN/v2alpha/sessions \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer '"$TOKEN"''\
+  --header 'Content-Type: application/json' \
+  --data '{
+  "checks": {
+    "user": {
+      "loginName": "minni-mouse@mouse.com@fabi.zitadel.app"
+    }
+  },
+  "metadata": {},
+  "challenges": [
+    "CHALLENGE_KIND_PASSKEY"
+  ]
+}'
+```
+
+### Response
+```bash
+{
+  "details": {
+    "sequence": "2",
+    "changeDate": "2023-06-27",
+    "resourceOwner": "69629023906488334"
+  },
+  "sessionId": "d654e6ba-70a3-48ef-a95d-37c8d8a7901a",
+  "sessionToken": "string",
+  "challenges": {
+    "passkey": {
+      "publicKeyCredentialRequestOptions": {
+        "publicKey": {
+          "allowCredentials": [
+            {
+              "id": "ATmqBg-99qyOZk2zloPdJQyS2R7IkFT7v9Hoos_B_nM",
+              "type": "public-key"
+            }
+          ],
+          "challenge": "GAOHYz2jE69kJMYo6Laij8yWw9-dKKgbViNhfuy0StA",
+          "rpId": "localhost",
```

Comment:
This does not align with the value above and could confuse people.

Let's settle on one domain to be used in the example

Reply:
changed